neon_utils/parse_utils.py

```diff
@@ -184,3 +184,43 @@ def normalize_string_to_speak(to_speak: str) -> str:
     if any(to_speak.endswith(x) for x in valid_punctuation):
         return to_speak
     return f"{to_speak}."
+
+
+def transliteration(transcription: str, text: str, lang: str) -> (str, str):
+    '''
+    Transliterates string from transcription provided by stt plugins
+    Transliterates string if transcribed string length equals filename text length
+    :param transcription: input text to transliterate
```

Comment:
Please add these for `text` and `lang` params. Make sure to note `lang` should be an ISO-639-1 code (i.e. "en")